### PR TITLE
Add instructions on install from Homebrew

### PR DESCRIPTION
**Pull request by DaniruKun**
There is now a [Homebrew formula](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/podman-compose.rb) for podman compose.

### CODE REVIEW

**Comment by DaniruKun** [comment]
@muayyad-alsadi  wdyt?

**Comment by DaniruKun** [comment]
@Evedel maybe you can take a look?

**Comment by Evedel** [comment]
Hey @DaniruKun, sorry, not a maintainer

**Comment by DaniruKun** [comment]
@muayyad-alsadi ?

File: README.md [hunk 1]
```diff
@@ -49,24 +49,45 @@ like `hostnet`. If you desire that behavior, pass it the standard way like `netw
...
 
-or install from Fedora (starting from f31) repositories:
+```bash
+brew install podman-compose
+```
+
+### Manual
+
+```bash
+curl -o /usr/local/bin/podman-compose https://raw.githubusercontent.com/containers/podman-compose/devel/podman_compose.py
```
Comment (p12tic) [inline README.md:77]: podman-compose switched to main branch. Hmm, existing documentation hasn't been updated unfortunately.

**Review by p12tic — APPROVED** [review]
Looks great, please rebase.

If the existing documentation still refers to devel branch when you rebase, could you please fix it in a separate commit that comes before your changes? Thanks!

**Comment by p12tic** [comment]
Rebased and addressed comments.

**Pull request merged**